crypto/bio/bss_dgram_pair.c

```diff
@@ -270,9 +270,9 @@ struct bio_dgram_pair_st {
     unsigned int grows_on_write    : 1; /* Set for BIO_s_dgram_mem only */
 };
 
```

Comment:
it would be nice to drop those changes here to decrease the clutter. those stylistic changes are not related to your addition of SSL_get_peer_addr(). thanks.

Reply:
I see, if you just revert those changes so the file will disappear from changeset the check should go green if not then we can wave the c-style. thanks.

Reply:
thanks. now the file is not listed in the changeset anymore.